metapool/sample_sheet.py

```diff
@@ -280,24 +280,34 @@ def write_blank_lines(writer, n=blank_lines, width=csv_width):
     def merge(self, sheets):
         """Merge the Data section of multiple sample sheets
 
+        For the Date field in the Header section, we only keep the date of the
```

Comment:
Line 281 says Data, here it says Date, are both correct?

Reply:
You are right, it is Date. Date is a field in the Header section and Data is a different section. Hierarchy is: Section > Field